### ISSUE TITLE
export DOCKER_BUILDKIT=1 is -required-

### ISSUE THREAD

**Issue by cryo** [dholth/vagrant-docker#1]
This should be in the readme.md and save someone else 30 minutes of their life.

**Comment by cryo** [comment]
line 19 ERROR: executor failed running [...] : exit code: 1
failed to solve...

So even though --mount works, that line dies for some reason that thankfully error code 1 will explain.

**Comment by dholth** [comment]
Glad you tried it. Did you get here from the blog post? See also https://github.com/dholth/vagrant-docker/blob/master/Dockerfile#L8

**Comment by cryo** [comment]
Yes came here from the blog... I think that one of the packages might not be available anymore in the list or something that is causing the error 1.  Are you able to repro it?  Also, it wasn't intuitive to see the comment in the dockerfile because I was just following the instructions verbatim.  I actually didn't even look there but found a stackexchange(!) comment on someone else having issues with the --mount command :).  I was also tired of the fans coming on, as you had mentioned... though the docker use of the abandoned hypervisor library that connects to the macOS hypervisor isn't great either because it eats CPU as well.  sigh.

**Comment by dholth** [comment]
Maybe the `--skip-broken` argument should be added to the yum install commands, and the example package list should be shorter since you're really supposed to generate it yourself after the first regular install.
